sonar-plugin-api/src/main/java/org/sonar/api/Plugin.java

```diff
@@ -85,18 +85,27 @@
 public interface Plugin {
 
   class Context {
-    private final Version version;
+    private final SonarRuntime sonarRuntime;
     private final List extensions = new ArrayList();
 
-    public Context(Version version) {
-      this.version = version;
+    public Context(SonarRuntime sonarRuntime) {
+      this.sonarRuntime = sonarRuntime;
     }
 
     /**
-     * Runtime version of SonarQube
+     * @deprecated since 6.0 use {@link #getRuntimeApiVersion()}
      */
+    @Deprecated
     public Version getSonarQubeVersion() {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1133)